src/pages/Login/Login.tsx

```diff
@@ -1,10 +1,23 @@
+import { useState } from 'react';
 import { useEffect } from 'react';
 import { useLoginStatus } from '../../hooks/LoginStatusContext';
 import './Login.css';
 import { useNavigate } from 'react-router-dom';
+
 export const Login = () => {
+	let [firstView, setFirstView] = useState(0);
 	const { getUserData, isLogged } = useLoginStatus();
-	const navigate = useNavigate();
+	const [loginError, setLoginError] = useState(false);
+	const navigate = useNavigate()
+
+	console.log(firstView, loginError)
+
+	useEffect(() => {
+		if (firstView) {
```

Comment:
You are checking firstView but not for the false state from the LoginStatus hook